main.rb

```diff
@@ -79,38 +85,31 @@ def create_teacher
     @people << new_teacher
     puts
   end
+end
 
-  def create_book
-    print 'Title: '
-    title = gets.chomp
-    print 'Author: '
-    author = gets.chomp
-    new_book = Book.new(title, author)
-    @books << new_book
-    puts 'New book created successfully'
-    puts
+class Rentals
```

Comment:
- [ ] Kindly move this **Rentals** class into a different file in keeping with the SOLID rule